src/ManageContainer/Client/ManageToDb/Client_test.go

```diff
@@ -342,5 +343,29 @@ func TestInsertModelParamsParallelRejectDuplicate(t *testing.T) {
 	initialize()
 }
 
+func TestGetElapsedTimeSuccess(t *testing.T) {
+	initialize()
+
+	os.Mkdir(fmt.Sprintf("/Db/result/%s", defaultJobUUID), 0777)
+	data := `{"id":"","job_uuid":"m2db_test_jobuuid","result":"[\"1\",\"2\",\"3\"]","meta":{"piece_id":0}}`
+	ioutil.WriteFile(fmt.Sprintf("/Db/result/%s/%d", defaultJobUUID, defaultPieceID), []byte(data), 0666)
+	clock_start := "1666942928259"
+	clock_end := "1666943026561"
+	os.Create(fmt.Sprintf("/Db/result/%s/completed", defaultJobUUID))
+	ioutil.WriteFile(fmt.Sprintf("/Db/result/%s/status_PRE_JOB", defaultJobUUID), []byte(clock_start), 0666)
+	ioutil.WriteFile(fmt.Sprintf("/Db/result/%s/status_COMPLETED", defaultJobUUID), []byte(clock_end), 0666)
+	expect := 98.302
+
+	client := Client{}
+	elapsedTime, err := client.GetElapsedTime(defaultJobUUID)
+
+	if err != nil {
+		t.Error("get elapsed time failed: " + err.Error())
+	}
+	if math.Abs(elapsedTime - expect) > 0.001 {
+		t.Errorf("get elapsed time failed: elapsed_time must be %f, but value is %f.",expect,elapsedTime)
```

Comment:
typo: elapsed_time -> elapsedTime
```suggestion
		t.Errorf("get elapsed time failed: elapsedTime must be %f, but value is %f.",expect,elapsedTime)
```

Reply:
@mdonaka 
Thank you for your review
Please check the correction.